servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -209,7 +211,7 @@ public EvictionPlan freeSpace(long bytesToBeAvailable, BlockStoreLocation locati
       }
     }
 
-    return EvictorUtils.legalCascadingPlan(bytesToBeAvailable, plan, mMetaManager) ? plan : null;
+    return EvictorUtils.validCascadingPlan(bytesToBeAvailable, plan, mMetaManager) ? plan : null;
```

Comment:
Question: is calling `EvictorUtils.validCascadingPlan` here just to verify the plan is correct, or it is required for the return value? If it is not required, we shall not have the verification on the critical code path---during eviction heavy lock is hold.